scripts/issue/836/deposit_badger_fraxbp_lp.py

```diff
@@ -0,0 +1,36 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from rich.console import Console
+
+console = Console()
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_convex()
+
+    lp = vault.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+
+    (_, _, _, rewards) = vault.convex.get_pool_info(lp)
+    rewards = vault.contract(rewards)
+    rewards_before = rewards.balanceOf(vault)
+
+    vault.take_snapshot(tokens=[lp.address])
+
+    # this doesn't return `True` which fails assertion check?
+    # vault.convex.deposit_all_and_stake(lp)
```

Comment:
works for me. maybe try a different rpc?

source code:
```
//deposit all lp tokens and stake
function depositAll(uint256 _pid, bool _stake) external returns(bool){
    address lptoken = poolInfo[_pid].lptoken;
    uint256 balance = IERC20(lptoken).balanceOf(msg.sender);
    deposit(_pid,balance,_stake);
    return true;
}
```

Reply:
tried multiple api keys using alchemy and infura as well as resetting `.brownie/deployments.db` and still get the same result.  Interestingly, calling `Booster.depositAll` works fine along with verifying the state before and after; the balance reflects what should be staked. It's just the return value that isn't playing nicely. would be nice to see if other reviewers could confirm if ` vault.convex.deposit_all_and_stake(lp)` is working for them

Reply:
it also does work for me locally.

```
Transaction sent: 0xea967ec02d5b118c7d884f1e1cc2e851912cf4031d0534e13298c2a0fc4c5db0
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  Booster.depositAll confirmed   Block: 15688048   Gas used: 540252 (4.50%)

Transaction sent: 0x1908045b6fd5abe3ac1f988516db103051ca6f508470a5a074e4d2bb9e90d130
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  Vyper_contract.approve confirmed   Block: 15688049   Gas used: 14733 (0.12%)

rewards_before=0
rewards_after=132176996931704087847121
delta=132176996931704087847121
```